crates/nu-command/src/bytes/index_of.rs

```diff
@@ -134,19 +131,19 @@ fn index_of(val: &Value, args: &Arguments, span: Span) -> Value {
     match val {
         Value::Binary {
             val,
-            span: val_span,
```

Comment:
for matching, look at the PR where we use the `.span()` to get the span out of the value rather than matching. You shouldn't match `internal_span`

Reply:
Oh, sorry for missing that, I've changed all internal_span matched to `.span()`.